src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -95,7 +95,6 @@ public static int LastIndexOfAny(ref byte searchSpace, int searchSpaceLength, re
         }
 
         // Adapted from IndexOf(...)
-        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
This was put in place as speculative startup optimization originally, to avoid generating big inefficient non-optimized Tier 0 code, only to re-jit it soon after that.

I think this concern should be gone with crossgen2.

Is there anything particular details that you are worried about? I think it should be pretty safe to remove these `AggressiveOptimization` attributes here.